Microsoft.Recognizers.Text.Number/Spanish/Extractors/IntegerExtractor.cs

```diff
@@ -10,66 +10,68 @@ public class IntegerExtractor : BaseNumberExtractor
         internal override sealed ImmutableDictionary<Regex, string> Regexes { get; }
 
         protected override sealed string ExtractType { get; } = Constants.SYS_NUM_INTEGER;
-        
+
         public const string HundredsNumberIntegerRegex = @"(cuatrocient[ao]s|trescient[ao]s|seiscient[ao]s|setecient[ao]s|ochocient[ao]s|novecient[ao]s|doscient[ao]s|quinient[ao]s|(?<!por\s+)(cien(to)?))";
 
-        public const string RoundNumberIntegerRegex = @"(mil|millones|mill[oó]n|billones|bill[oó]n|trillones|trill[oó]n|cuatrillones|cuatrill[oó]n|quintillones|quintill[oó]n|sextillones|sextill[oó]n|septillones|septill[oó]n)";
+        public const string RoundNumberIntegerRegex = @"(mil millones|mil|millones|mill[oó]n|billones|bill[oó]n|trillones|trill[oó]n|cuatrillones|cuatrill[oó]n|quintillones|quintill[oó]n|sextillones|sextill[oó]n|septillones|septill[oó]n)";
```

Comment:
Nope, "mil millones" in Spanish maps to "billions" we need to have both in place ("mil millones" and "mil" otherwise "mil" will be hit instead of "mil millones".